CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
   places" layer. Scale point markers for this layer by population.
 * Add new "Internet-required data/Geological Map (1:500 000)" layer from GEUS.
 * Add new "Geology/Mineral occurrences" layer from GEUS.
+* Remove OBE references to "gdal_remote" layers. Remove "Online" from `QgsLayerType`.
```

Comment:
I think maybe we should omit this from the changelog. This will only make sense to us. Maybe we should have a devlog and changelog? :shrug:

Reply:
Yeah, I was debating this myself. We have tended to point users to our CHANGELOG, so removing this makes sense.